spec/models/reservation_spec.rb

```diff
@@ -921,6 +922,7 @@
       before :each do
         @rule.destroy
         @instrument.update_attribute :reserve_interval, 15
+        @instrument.reload
```

Comment:
Same with the rest of these, although, it could also be better to say `@instrument.schedule_rules.destroy_all` instead of `@rule.destroy`.